src/libraries/System.Private.CoreLib/src/System/Text/CompositeFormat.cs

```diff
@@ -0,0 +1,353 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace System.Text
+{
+    /// <summary>Represents a parsed composite format string.</summary>
+    [DebuggerDisplay("{Format}")]
+    public sealed class CompositeFormat
+    {
+        /// <summary>Number of chars to stackalloc as part of a formatting and parsing operations.</summary>
+        internal const int StackAllocCharSize = 256;
```

Comment:
>internal const int StackAllocCharSize = 256; [](http://example.com/codeflow?start=8&length=44)

why defined that and not used `string.StackallocCharBufferSizeLimit`?

Reply:
Oops. I added the centralized one on string to replace this and then neglected to remove this.